Code/Source/RobotImporter/Utils/RobotImporterUtils.h

```diff
@@ -10,22 +10,46 @@
 #include "AzCore/Component/ComponentBus.h"
 #include "AzCore/std/string/string.h"
 #include "RobotImporter/URDF/UrdfParser.h"
+#include <AzCore/IO/SystemFile.h>
 #include <AzCore/Math/Transform.h>
 #include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/std/function/function_template.h>
+
 namespace ROS2
 {
+    namespace
+    {
+        const AZStd::function<bool(const AZStd::string&)> fileExistsCall = [](const AZStd::string& fn) -> bool
```

Comment:
expand fn name to "filename" or such, fn could also mean "function" or something else